tests/yast2_gui/yast2_storage_ng.pm

```diff
@@ -0,0 +1,242 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: YaST2 ...
```

Comment:
in worst case copy/paste PR title here :)

Reply:
Good catch! I completely forgot about it :)